build.gradle.kts

```diff
@@ -48,6 +50,12 @@ allprojects {
             ktlint()
         }
     }
+
+    dependencies {
+        testCompile("com.google.truth:truth:+") {
```

Comment:
is this needed for this PR? if not, maybe include it in the next one when you add testing infra

Reply:
agree, that's for the next one.